.gitignore

```diff
@@ -1,3 +1,9 @@
+# Downloaded model files
+onnxt5/model_data
```

Comment:
If you `pip install -e .` and run this library, you need to ignore the path it downloads model files to.